app/models/manageiq/providers/ibm_cloud/cloud_object_storage/storage_manager/refresh_worker.rb

```diff
@@ -0,0 +1,3 @@
+class ManageIQ::Providers::IbmCloud::CloudObjectStorage::StorageManager::RefreshWorker < MiqEmsRefreshWorker
+  require_nested :Runner
+end
```

Comment:
Believe you'll need to set the `def self.settings_name` here like in the `vpc/cloud_manager/refresh_worker` class